pokemon-react-app/src/components/PokemonComponentList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { Ball } from '../styles/Ball.style'
+import { Img } from '../styles/Img.style'
+import { Label } from '../styles/Label.style'
+import { StyledLink } from '../styles/Link.style'
+
+function PokemonComponentList() {
+    const pokemons = useSelector(state => state.pokemons.pokemons)
+    const filteredPokemons = useSelector(state => state.pokemons.filteredPokemons)
+    const filterStatus = useSelector(state => state.pokemons.filterStatus)
+    var renderList=[]
+    if(pokemons.length!=0 && filteredPokemons.length==0 && filterStatus==false){
```

Comment:
Also please check here usage of double or triple equal, there are console warnings in browser in multiple places